.eslintrc.js

```diff
@@ -0,0 +1,10 @@
+const config = {
+	root: true,
+	extends: ['./node_modules/@talend/scripts-config-eslint/.eslintrc'],
+};
+
+if (process.argv.join(' ').includes('.ts')) {
```

Comment:
sorry I didn't find other way to fix husky call to eslint to avoid --no-verify which sometimes can be useful.